src/deluge/gui/views/instrument_clip_view.cpp

```diff
@@ -2816,27 +2821,32 @@ void InstrumentClipView::setRowProbability(int32_t offset) {
 	if (!noteRow) {
 		return; // Get out if NoteRow doesn't exist and can't be created
 	}
-	Action* action = actionLogger.getNewAction(ACTION_NOTE_EDIT, true);
-	if (!action) {
-		return;
-	}
-
-	action->recordNoteArrayChangeIfNotAlreadySnapshotted((InstrumentClip*)modelStack->getTimelineCounter(),
-	                                                     modelStackWithNoteRow->noteRowId, &noteRow->notes,
-	                                                     false); // Snapshot for undoability. Don't steal data.
 
 	uint8_t probabilityValue = noteRow->probabilityValue;
-	bool prevBase = false;
-	// Covers the probabilities and iterations
-	probabilityValue = std::clamp<int32_t>((int32_t)probabilityValue + offset, (int32_t)0,
-	                                       kNumProbabilityValues + kNumIterationValues);
 
-	noteRow->probabilityValue = probabilityValue;
+	// If editing, continue edit
+	if (display->hasPopup()) {
```

Comment:
Doesn't work - the display always has a popup when audition is held

Reply:
Yeah I know but I don't know how t bypass that. Ideas?

Reply:
Replaced the check of display->hasPopup with a boolean variable check, which is reset at the same time you stop auditioning a row